push/tests/test_views.py

```diff
@@ -7,8 +7,7 @@
 
 from ..forms import PushAppForm
 from ..models import PushApplication, MessagesAPIError
-from ..views import (Landing, PushApplications, Details,
-                     ValidatePushApplication, UserOwnsPushAppMixin)
+from ..views import Landing, List, Details, Validation, UserOwnsPushAppMixin
```

Comment:
The whole issue was secretly just a way to get this import to fit on a single line.